File: app/src/main/java/io/github/freewebmovement/igniter/persistence/database/EnabledPackage.java
package io.github.freewebmovement.igniter.persistence.database;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "enabled_packages")
public class EnabledPackage {
    @PrimaryKey
    @ColumnInfo(name = "package_name")
    @NonNull
    public String packageName;

    public EnabledPackage() {
        packageName = "";
    }
}

File: app/src/main/java/io/github/freewebmovement/igniter/persistence/database/EnabledPackage.kt
package io.github.freewebmovement.igniter.persistence.database

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "enabled_packages")
class EnabledPackage {
    @PrimaryKey
    @ColumnInfo(name = "package_name")
    var packageName: String = ""

    constructor()
}
